OCP 4.14: IBM PowerVS IPI updates for PER enablement (#65856)
openshift/openshift-docs: modules/quotas-and-limits-ibm-power-vs.adoc

There is a limit of two cloud connections per {ibmpowerProductName} Virtual Server instance. It is recommended that you have only one cloud connection in your {ibmpowerProductName} Virtual Server instance to serve your cluster.

[NOTE]
====

> Cloud Connections are no longer supported in `dal10`.

Reviewer: I might word it as
```
Cloud Connections are no longer supported in `dal10`. A transit gateway will be used instead.
```

Author: Ack. Updated the PR.
@mjturek

> Cloud Connections are no longer supported in `dal10`. A transit gateway will be used instead.

Reviewer 2: ```suggestion
Cloud connections are no longer supported in `dal10`. A transit gateway is used instead.
```

Reviewer 2: Per IBM Style, present tense is preferred.
